framework/source/class/qx/ui/embed/Iframe.js

```diff
@@ -335,7 +380,11 @@ qx.Class.define("qx.ui.embed.Iframe",
 
       if (iframeSource != this.getSource())
       {
-        qx.bom.Iframe.getWindow(iframeDomElement).stop();
+        if ( qx.core.Environment.get("browser.name") != "edge" &&
+             qx.core.Environment.get("browser.name") != "ie" )
+        {
+          qx.bom.Iframe.getWindow(iframeDomElement).stop();
+        }
```

Comment:
> Ist this stop() needed for all browsers except ie/edge?

To be honest, I don't think so.
But as I have no information about the initial need for this whole work-around I decided to keep it (as much as possible).

In my test(s) it makes no difference, but these test(s) only loaded small HTML pages ...
... I can imagine however, that if the first ("_pre-DOM-move_") URL loads lots of data it can make sense to stop loading that, when a "_DOM-move-and-URL-change_" was detected.
It might still be superfluous (as the URL is changed and the Client might stop loading the old URL anyway), but that's something I can neither confirm nor deny.